src/script.js

```diff
@@ -0,0 +1,39 @@
+const bscript = require('bscript-parser')
+
+class Script {
+  get length () {
+    return this.serialized.length
+  }
+
+  constructor (serializedScript) {
+    if (!Buffer.isBuffer(serializedScript)) {
```

Comment:
Do you think this should instead be a token array from the `bscript-parser` library?

Reply:
That's a good question. My initial vision of these projects was the `bscript-parser` was mainly to assemble/disassemble, for convenience and display. And this `bitcoin-script-js` module would be an alternative implementation of the script interpreter in Bitcoin Core, for example.  Since we're already using `bscript-parser` for `opcodeIsValid`, for example, it seems that it can serve as a helper library.

I'm not really sure if this should require a token array, since it is not likely familiar to users of this library, it might just be adding another barrier to entry.